gdk4/src/auto/enums.rs

```diff
@@ -621,6 +621,8 @@ pub enum EventType {
     PadStrip,
     #[doc(alias = "GDK_PAD_GROUP_MODE")]
     PadGroupMode,
+    #[doc(alias = "GDK_TOUCHPAD_HOLD")]
+    TouchpadHold,
```

Comment:
It probably needs a version guard

Reply:
Which version though? 4.6.1? 4.8?

Reply:
4.8 i guess

Reply:
It also needs to be added to https://github.com/gtk-rs/gtk4-rs/blob/master/gdk4/src/touchpad_event.rs#L9

Reply:
Conditionally though. I'll try :)